Testing/integration/VM/setup.sh

```diff
@@ -33,16 +33,23 @@ fi
 # Install rosetta (for test binaries)
 softwareupdate --install-rosetta --agree-to-license
 
-# Install actions runner
-mkdir ~/actions-runner
-pushd ~/actions-runner
-curl -o actions-runner-osx-arm64-2.296.0.tar.gz -L https://github.com/actions/runner/releases/download/v2.296.0/actions-runner-osx-arm64-2.296.0.tar.gz
-echo 'e358086b924d2e8d8abf50beec57ee7a3bb0c7d412f13abc51380f1b1894d776  actions-runner-osx-arm64-2.296.0.tar.gz' | shasum -a 256 -c
-tar xzf ./actions-runner-osx-arm64-2.296.0.tar.gz
-./config.sh --url https://github.com/google/santa
-./svc.sh install
-./svc.sh start
-popd
+# Add a LaunchAgent to start the mounted runner
+cat | sudo tee ${HOME}/Library/LaunchAgents/runner.plist << EOF
```

Comment:
```suggestion
sudo tee ${HOME}/Library/LaunchAgents/runner.plist <<EOF
```

But also: why is sudo used? ~/Library/LaunchAgents is always writeable by the owning user.

Reply:
Was originally putting this in /System - leftover from that. Changed.